pkgs/top-level/all-packages.nix

```diff
@@ -13917,8 +13917,6 @@ with pkgs;
 
   verilator = callPackage ../applications/science/electronics/verilator { };
 
-  verilog = callPackage ../applications/science/electronics/verilog { };
```

Comment:
Please add an alias in top-level/aliases.nix

Reply:
Done.